platform/eureka/testdata/eureka-apps.json

```diff
@@ -0,0 +1,114 @@
+{
+   "applications": {
+      "versions__delta": "1",
+      "apps__hashcode": "UP_34_",
+      "application": [
+         {
+            "name": "FOO_BAR_LOCAL",
+            "instance": [
+               {
+                  "instanceId": "foo.bar.local-10.0.0.1-5000",
+                  "hostName": "foo.bar.local",
+                  "app": "FOO_BAR_LOCAL",
+                  "ipAddr": "10.0.0.1",
+                  "status": "UP",
+                  "overriddenstatus": "UNKNOWN",
+                  "port": {
+                     "$": 5000,
+                     "@enabled": "true"
+                  },
+                  "securePort": {
+                     "$": 5443,
+                     "@enabled": "true"
+                  },
+                  "countryId": 1,
+                  "leaseInfo": {
+                     "renewalIntervalInSecs": 30,
+                     "durationInSecs": 90,
+                     "registrationTimestamp": 1502120007390,
+                     "lastRenewalTimestamp": 1502120127413,
+                     "evictionTimestamp": 0,
+                     "serviceUpTimestamp": 1502120006871
+                  },
+                  "metadata": {
+                     "istio.protocol": "HTTP"
+                  },
+                  "isCoordinatingDiscoveryServer": "false",
+                  "lastUpdatedTimestamp": "1502120007390",
+                  "lastDirtyTimestamp": "1502120006858",
+                  "actionType": "ADDED"
+               },
+               {
+                  "instanceId": "foo.bar.local-10.0.0.2-6000",
+                  "hostName": "foo.bar.local",
+                  "app": "FOO_BAR_LOCAL",
+                  "ipAddr": "10.0.0.2",
+                  "status": "UP",
```

Comment:
This normally indicates health as posted by local healthchecks or those embedded in clients like ribbon. Since this is updated by each application instance, I think we should filter out instances with status: "down".